src/portable/raspberrypi/rp2040/hcd_rp2040.c

```diff
@@ -294,7 +294,6 @@ static void _hw_endpoint_init(struct hw_endpoint *ep, uint8_t dev_addr, uint8_t
     uint8_t const num = tu_edpt_number(ep_addr);
     tusb_dir_t const dir = tu_edpt_dir(ep_addr);
 
-    bool in = ep_addr & TUSB_DIR_IN_MASK;
```

Comment:
good catch, apparently we haven't enabled all warnings like other builds.

Reply:
yeah it builds with warnings in our SDK test

Reply:
it should cause the build to failed, I normally have `-Werror -Wfatal-errors` in other gnu make build. Will figure out how to enable this later on.